src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -238,7 +238,7 @@ const SummaryBar = props => {
             </Row>
           </Col>
 
-          <Col className="col-lg-3 col-12 no-gutters">
+          <Col className="d-flex col-lg-3 col-12 no-gutters">
```

Comment:
Hi @calexander3108, 

I added d-flex to 241 as well because without that then when the window becomes smaller the summary columns does not fill out the height. This only happens when the user name is long. Please see image below for example. Thanks! 

![Screenshot 2023-03-18 at 3 53 02 PM](https://user-images.githubusercontent.com/97488347/226144367-157ac999-6f69-43c3-af40-1a564ff4363e.png)

Reply:
got it thanks for the example @angelinatr15! Looks great!